web/core/components/sidebar/sidebar-navigation.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+import React, { FC } from "react";
+// helpers
+import { cn } from "@/helpers/common.helper";
+
+type TSidebarNavigation = {
+  label: string | React.ReactNode;
+  className?: string;
+  isActive?: boolean;
+  icon?: React.ReactNode;
+  indicatorElement?: React.ReactNode;
+};
+
+export const SidebarNavigation: FC<TSidebarNavigation> = (props) => {
```

Comment:
Use a different name. This sounds like an action.

Reply:
Resolved